solutions/standard/variables.tf

```diff
@@ -60,6 +60,12 @@ variable "backup_crn" {
   default     = null
 }
 
+variable "remote_leader_crn" {
+  type        = string
+  description = "A CRN of the leader database to make the replica(read-only) deployment. The leader database is created by a database deployment with the same service ID. A read-only replica is set up to replicate all of your data from the leader deployment to the replica deployment by using asynchronous replication. [Learn more](https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-read-only-replicas)."
```

Comment:
@MatthewLemmond FYI i pushed a commit to update the description of the DA variable. Instead of adding a long url to the description, which will look horrible in projects UI, use a markdown link as that will render in projects UI.